static/js/components/ResumeLinkedIn.js

```diff
@@ -25,6 +29,17 @@ import type {
   ResumeLinkedInResponse
 } from "../flow/applicationTypes"
 
+const defaultLinkedInErrorMsg = (
+  <span>
+    Something went wrong while adding your LinkedIn profile. Please try again,
+    or{" "}
+    <a href={SETTINGS.support_url} target="_blank" rel="noopener noreferrer">
+      contact support
+    </a>{" "}
+    to resolve this issue.
```

Comment:
This part of the error message is repeated in a few places, maybe put it in a util function?

Reply:
Makes sense